scm-info/org/mvnpm/_artifact/lit/scm.yaml

```diff
@@ -3,4 +3,8 @@ type: "git"
 uri: "https://github.com/lit/lit.git"
 tagMapping:
   - pattern: 2.6.1
-    tag: lit@2.6.1
\ No newline at end of file
+    tag: lit@2.6.1
```

Comment:
I will merge, but shouldn't this be a regex pattern rather than an explicit list (same with the one above)?

Reply:
Agreed, let me make a PR for the same post testing

Reply:
I tried using below but it does not work. Tested with normal regex checkers, but it fails while running with this repository
```
tagMapping:
  - pattern: (\d+)\.(\d+)\.(\d+)
    tag: lit\@$1\.$2\.$3
```